app-authz-jakarta-servlet/config/realm-import.json

```diff
@@ -82,11 +80,11 @@
     {
       "clientId": "authz-servlet",
       "enabled": true,
-      "baseUrl": "http://localhost:8080/authz-servlet",
-      "adminUrl": "http://localhost:8080/authz-servlet",
+      "baseUrl": "http://127.0.0.1:8080/authz-servlet",
+      "adminUrl": "http://127.0.0.1:8080/authz-servlet",
       "bearerOnly": false,
       "redirectUris": [
-        "http://localhost:8080/authz-servlet/*"
+        "*"
```

Comment:
I wonder why is this needed? We mention in the "Threat Docs" https://www.keycloak.org/docs/latest/server_admin/index.html#open-redirectors to avoid wildcards, so does not look very great to use them in the quickstart?

Also is it strictly needed to change `baseUrl` and `adminUrl` to `127.0.01` instead of `localhost`? Is it just because of the tests or any other reason for this?